PowerballApi/App_Start/Bootstrapper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Web.Http;
+using Autofac;
+using Autofac.Integration.WebApi;
+using PowerballApi.Api.Helpers.Cacher;
+using PowerballApi.Api.Helpers.HttpHandler;
+using PowerballApi.Api.Helpers.Parser;
+using PowerballApi.Api.Models;
+using PowerballApi.Api.Repositories;
+using PowerballApi.Api.Services;
```

Comment:
You probably want to put these using statements within the scope of the namespace. This keeps somewhat rare but difficult to pinpoint conflicts from happening.

Reply:
updated!